api/types.go

```diff
@@ -115,3 +115,108 @@ type TaggedMetric struct {
 
 // GraphiteMetric is a flat, dot-separated identifier to a series of metric.
 type GraphiteMetric string
+
+// Column is a different aspect of data.
+// For example, Blueflood may stores (min / max / average / count) during rollups,
+// and these data are exposed via columns
+type Column string
```

Comment:
Sounds like this should be something like a SeriesType? Column seems too generic and backend-aware.